ProjectUpdater/src/au/gov/asd/tac/projectupdater/ProjectUpdater.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.projectupdater;
```

Comment:
You should update the package names.

Reply:
to `au.gov.asd.tac.constellation.projectupdater`  @cygnus-x-1 ?

Reply:
Updated it @cygnus-x-1